fsharp-backend/tests/httpbytestestfiles/simple-request-headers.test

```diff
@@ -0,0 +1,41 @@
+[http-bytes-handler GET /]
+Http.response (request.headers |> toString |> String.toBytes_v0) 200
+
+[request]
+GET / HTTP/1.1
+Host: HOST
+Date: Sun, 08 Nov 2020 15:38:01 GMT
+Content-Length: 7
+header1: value1
+Header2: Value2
+Header-3: Value-3
+Header-with-Casing-conflict: test1
+header-WITH-Casing-conflict: test2
+
+ignored
+
+[response]
+HTTP/1.1 200 OK
+Date: xxx, xx xxx xxxx xx:xx:xx xxx
+x-darklang-execution-id: 0123456789
+Server: darklang
+Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
+Content-Length: 352
+
+[<SPACE>
+  (
+    "host", "HOST"
+  ), (
+    "date", "Sun, 08 Nov 2020 15:38:01 GMT"
+  ), (
+    "content-length", "7"
+  ), (
+    "header1", "value1"
+  ), (
+    "header2", "Value2"
+  ), (
+    "header-3", "Value-3"
+  ), (
+    "header-with-casing-conflict", "test1,test2"
```

Comment:
I don't understand why this isn't two headers. Kind of the point of using a list of tuples here, is to allow for multiple headers of the same name.

Reply:
it was using a `getHeaders` in Server.fs fn that concatonated the values; fixed